tiledb/api/src/filter/list.rs

```diff
@@ -183,75 +180,6 @@ impl Builder {
     }
 }
 
-#[derive(
-    Clone, Debug, Default, Deserialize, OptionSubset, PartialEq, Serialize,
-)]
-pub struct FilterListData(Vec<FilterData>);
```

Comment:
In contrast to other items which are moved somewhere else, this one I just deleted and replaced uses of it with `Vec<FilterData>`.